pcdsdevices/attenuator.py

```diff
@@ -1071,6 +1072,114 @@ class AT2L0(FltMvInterface, PVPositionerPC, LightpathInOutMixin):
     # Summary for lightpath view
     num_in = Cpt(InternalSignal, kind='hinted')
     num_out = Cpt(InternalSignal, kind='hinted')
+    
+    """check for errors and return a string indicating any errors verbally"""
+    def _check_errors(signals: SignalToValue) ->  str:
+        errors = []
+        #sort out .motor from .motor.plc signals
+        for sig, value in signals.items():
+            if ("motor" in sig.name) ^ ("plc" in sig.name):
+                value=sig.metadata.get('severity').value
```

Comment:
The formatting indicates you didn't get install pre-commit. I commented in your other PR but will mention it here in case you read it first: https://confluence.slac.stanford.edu/display/PCDS/How+to+add+a+device+to+pcdsdevices#Howtoaddadevicetopcdsdevices-Pre-commit

Also, this might be preferable:

```suggestion
                value = int(sig.metadata["severity"])
```

For future reference, too, `IntEnum` behaves like an integer here. We can compare it directly:

```
>>> at2l0.blade_01.state.state.metadata["severity"] in (0, "")
True
```